src/data/nsynth.py

```diff
@@ -17,15 +17,15 @@ def nsynth_from_tfrecord(nsynth_tfrecord_path):
 
 def nsynth_to_melspec(dataset, hparams):
     if 'instrument' in hparams:
```

Comment:
Having behavior depend on the existence of a config key doesn't sound user-friendly, and makes it impossible to override with a commandline flag. (Commandline flags can't set it to a non-string either, though, so this isn't ideal either.)
```suggestion
    if hparams['instrument'] is not None:
```